src/robot/libraries/XML.py

```diff
@@ -1378,7 +1492,12 @@ def element_to_string(self, source, xpath=".", encoding=None):
             string = string.encode(encoding)
         return string
 
-    def log_element(self, source, level="INFO", xpath="."):
+    def log_element(
+        self,
+        source: Any,
+        level: logger.LogLevel = "INFO",
+        xpath: str = ".",
+    ) -> str:
```

Comment:
Looking at the code, this method may actually return the value as `bytes`. Need to check that and either change implementation or typing.